### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by ph0ph0**
Updates the "Install a specific version" section with the correct command

Closes #???

## Description

Updates the docs to include the new version specific install command, as discussed here:
https://github.com/onflow/flow-cli/issues/737#issuecomment-1353036630

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Review by sideninja — CHANGES_REQUESTED** [review]
If installing a version before v0.42.0 you need to reference a section below in the docs. So changing that from master to an outdated hash is not a solution. I can agree that the version in the example should be after v0.42.0 tho.

**Comment by ph0ph0** [comment]
Ok cool, thanks for the info @sideninja, I have updated the PR!

File: docs/install.md [hunk 1]
```diff
@@ -42,12 +42,14 @@ To update, simply re-run the installation command above.
 
 ### Install a specific version
 
-To install a specific version of Flow CLI, append the version tag to the command (e.g. the command below installs CLI version v0.33.3).
+To install a specific version of Flow CLI newer than v0.42.0, append the version tag to the command (e.g. the command below installs CLI version v0.44.0).
 
 ```sh
-sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)" -- v0.33.3
+sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/50058df8aa5f999a85cfd7afae6fc2661090078a/install.sh)" -- v0.44.0
```
Comment (sideninja) [inline docs/install.md:48]: this should be kept `master` for new versions

Comment (ph0ph0) [reply docs/install.md]: Apologies, forgot to switch it back! I've made the change :)

**Review by sideninja — APPROVED** [review]
Thank you.

**Pull request merged**